specification/eventgrid/data-plane/Microsoft.EventGrid/stable/2018-01-01/GeneratedSystemEvents.json

```diff
@@ -254,6 +445,53 @@
         }
       ]
     },
+    "AcsChatAzureBotCommandReceivedInThreadEventData": {
```

Comment:
shouldn't the schema look the same between chat typing indicator and chat azure bot command? They seem to be including different set of properties. one has metadata, other does not. Can you double check this?